sycl/doc/extensions/proposed/sycl_ext_oneapi_usm_malloc_properties.asciidoc

```diff
@@ -0,0 +1,1350 @@
+   const propertyListA &propList)
+----
+a@ 
+Returns an `annotated_ptr` containing a raw pointer to the newly allocated memory of type specified by the `sycl::ext::oneapi::experimental::usm_kind` property.
+This allocation is specified in bytes.
+
+`propList` must contain the `sycl::ext::oneapi::experimental::usm_kind` property, otherwise the compiler will issue a diagnostic error.
+Additional runtime and compile-time constant properties can be provided in `propList`.
+The compile-time constant properties in `propList` will also appear as properties of the returned `annotated_ptr`.
+Runtime properties in `propList` will not appear as properties of the returned `annotated_ptr`.
+
+The `syclDevice` parameter is
+ignored if the allocation kind is `sycl::usm::alloc::host`.  If the allocation kind is not
+`sycl::usm::alloc::host`, `syclDevice` must either be contained by
+`syclContext` or it must be a descendent device of some device that
+is contained by that context, otherwise this function throws a synchronous
+`exception` with the `errc::invalid` error code.
+
+Available only if `propertyListA` and `propertyListB` are specializations of the `properties` class.
+
+a@
+[source]
+----
+template <typename T,
+          typename propertyListA,
+          typename propertyListB>
+annotated_ptr<T, propertyListB>
+malloc_annotated(
+   size_t count,
+   const device& syclDevice,
+   const context& syclContext,
+   const propertyListA &propList)
+----
+a@ 
+Returns an `annotated_ptr` containing a raw pointer to the newly allocated memory of type specified by the `sycl::ext::oneapi::experimental::usm_kind` property.
+This allocation is specified in number of elements of type `T`.
+
+`propList` must contain the `sycl::ext::oneapi::experimental::usm_kind` property, otherwise the compiler will issue a diagnostic error.
+Additional runtime and compile-time constant properties can be provided in `propList`.
+The compile-time constant properties in `propList` will also appear as properties of the returned `annotated_ptr`.
+Runtime properties in `propList` will not appear as properties of the returned `annotated_ptr`.
+
+The `syclDevice` parameter is
+ignored if the allocation kind is `sycl::usm::alloc::host`.  If the allocation kind is not
+`sycl::usm::alloc::host`, `syclDevice` must either be contained by
+`syclContext` or it must be a descendent device of some device that
+is contained by that context, otherwise this function throws a synchronous
+`exception` with the `errc::invalid` error code.
+
+Available only if `propertyListA` and `propertyListB` are specializations of the `properties` class.
+
+a@
+[source]
+----
+template <typename propertyListA, typename propertyListB>
+annotated_ptr<void, propertyListB>
+malloc_annotated(
+   size_t numBytes,
+   const queue& syclQueue,
+   const propertyListA &propList)
+----
+a@ Simplified form where `syclQueue` provides the `context`
+and any necessary `device`.
+
+a@
+[----
+source]
```

Comment:
I think this is some sort of typo that got committed by mistake.

Reply:
Thanks for catching this. Fixed.